include/wx/animate.h

```diff
@@ -38,9 +38,9 @@ class WXDLLIMPEXP_CORE wxAnimation : public wxObject
     wxAnimation();
     explicit wxAnimation(const wxString &name, wxAnimationType type = wxANIMATION_TYPE_ANY);
 
-#if __cplusplus >= 201103
-    wxAnimation(const wxAnimation&) = default;
-    wxAnimation& operator=(const wxAnimation&) = default;
+#ifdef wxMEMBER_DEFAULT
```

Comment:
I'd like to write this as
```suggestion
#ifdef wxHAS_MEMBER_DEFAULT
```
for consistency with the other similar macros. Also, by a coincidence, I've just added `wxHAS_NOEXCEPT` and `wxNOEXCEPT` in #1788, and it would really make sense to handle them in the same way.

Reply:
Maybe I can remove `wxHAS_MEMBER_DEFAULT` entirely? Guarding the function declaration does not generate build warnings or errors, only an Intellisense warning about 'function definition not found'.

Reply:
This would definitely generate link errors if either of the special methods is actually used with pre-C++11 compiler.

This is the problem, `wxMEMBER_DEFAULT` and `wxMEMBER_DELETE` are not symmetrical: the latter can be used unconditionally, but the former really can't be.